src/main/java/slimeknights/tconstruct/world/TinkerWorld.java

```diff
@@ -126,113 +127,114 @@ public TinkerWorld() {
+    ItemRegistryAdapter registry = new ItemRegistryAdapter(event.getRegistry());
     CreativeTab tabWorld = TinkerRegistry.tabWorld;
 
-    registerBlockItem(registry, green_slime_dirt, tabWorld);
-    registerBlockItem(registry, blue_slime_dirt, tabWorld);
-    registerBlockItem(registry, purple_slime_dirt, tabWorld);
-    registerBlockItem(registry, magma_slime_dirt, tabWorld);
-
-    registerBlockItem(registry, blue_vanilla_slime_grass, tabWorld);
-    registerBlockItem(registry, purple_vanilla_slime_grass, tabWorld);
-    registerBlockItem(registry, orange_vanilla_slime_grass, tabWorld);
-    registerBlockItem(registry, blue_green_slime_grass, tabWorld);
-    registerBlockItem(registry, purple_green_slime_grass, tabWorld);
-    registerBlockItem(registry, orange_green_slime_grass, tabWorld);
-    registerBlockItem(registry, blue_blue_slime_grass, tabWorld);
-    registerBlockItem(registry, purple_blue_slime_grass, tabWorld);
-    registerBlockItem(registry, orange_blue_slime_grass, tabWorld);
-    registerBlockItem(registry, blue_purple_slime_grass, tabWorld);
-    registerBlockItem(registry, purple_purple_slime_grass, tabWorld);
-    registerBlockItem(registry, orange_purple_slime_grass, tabWorld);
-    registerBlockItem(registry, blue_magma_slime_grass, tabWorld);
-    registerBlockItem(registry, purple_magma_slime_grass, tabWorld);
-    registerBlockItem(registry, orange_magma_slime_grass, tabWorld);
-
-    registerBlockItem(registry, blue_slime_leaves, tabWorld);
-    registerBlockItem(registry, purple_slime_leaves, tabWorld);
-    registerBlockItem(registry, orange_slime_leaves, tabWorld);
-
-    registerBlockItem(registry, blue_slime_fern, tabWorld);
-    registerBlockItem(registry, purple_slime_fern, tabWorld);
-    registerBlockItem(registry, orange_slime_fern, tabWorld);
-
-    registerBlockItem(registry, blue_slime_tall_grass, tabWorld);
-    registerBlockItem(registry, purple_slime_tall_grass, tabWorld);
-    registerBlockItem(registry, orange_slime_tall_grass, tabWorld);
-
-    registerBlockItem(registry, blue_slime_sapling, tabWorld);
-    registerBlockItem(registry, orange_slime_sapling, tabWorld);
-    registerBlockItem(registry, purple_slime_sapling, tabWorld);
-
-    registerBlockItem(registry, purple_slime_vine, tabWorld);
-    registerBlockItem(registry, purple_slime_vine_middle, tabWorld);
-    registerBlockItem(registry, purple_slime_vine_end, tabWorld);
-
-    registerBlockItem(registry, blue_slime_vine, tabWorld);
-    registerBlockItem(registry, blue_slime_vine_middle, tabWorld);
-    registerBlockItem(registry, blue_slime_vine_end, tabWorld);
+    SpawnEggItem blueSlimeSpawnEgg = new SpawnEggItem(WorldEntities.blue_slime_entity, 0x47eff5, 0xacfff4, (new Item.Properties()).group(ItemGroup.MISC));
+    registry.register(blueSlimeSpawnEgg, "blue_slime_spawn_egg");
+
+    registry.registerBlockItem(green_slime_dirt, tabWorld);
```

Comment:
Actually considering if I don't make it so that the tab also is saved inside the ItemRegistryAdapter